src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -147,10 +158,19 @@ class ReportActionContextMenu extends React.Component {
         return lodashGet(message, 'text', '');
     }
 
+    deleteAccepted() {
```

Comment:
Missing method doc. Also, NAB but I think it makes more sense to call this `confirmDeleteAndHideModal` or something to conform to our code style regulation that methods should be named for what they do, not the event they handle.